app/assets/sass/application.scss

```diff
@@ -28,3 +28,49 @@ $path: "/public/images/";
 }
 
 // Add extra styles here, or re-organise the Sass files in whichever way makes most sense to you
+
```

Comment:
Styling is stolen from https://github.com/aduggin/govuk-accessibility-sandbox/blob/master/app/assets/sass/components/_autocomplete.scss, with slight modifications.